### DISCUSSION TITLE
cn/2020/11/network-representation-measures-and-metrics/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#25] [Comments]
# cn/2020/11/network-representation-measures-and-metrics/

本文为《复杂网络系列》文章 本文内容主要参考自：《网络科学引论》1 网络（network）也称为图（graph），是一个由多个顶点（vertex

https://leovan.me/cn/2020/11/network-representation-measures-and-metrics/

**Comment by XinWang-HNU** [comment]
简洁明了，最佳博客。
